src/index.html

```diff
@@ -96,6 +96,12 @@ <h3 id="logo">Logo</h3>
                 <img class="img-responsive thumbnail-primary-bg" alt="" src="images/logo/text-white-320px.png">
             </div>
         </div>
+        <!-- Vector-Logos ================================================== -->
+        <div class="row">
+            <div class="col-lg-2 bs-component">
+                <img class="img-responsive thumbnail" alt="Cloudogu Logo" src="images/logo/blib-blue-vec.svg">
```

Comment:
Good catch to add it.

Studying `index.html` I find above this usage way more PNG versions as example. Would it make sense to prefer the SVG version in any way?

One thing could be to move the vector logo above the PNG logo instances or to replace the PNGs with the SVG altogether.

Reply:
Most images are currently unsused. I added the white logo as .svg as this sees some use as well. Further logo-images may be added as need arises